packages/orama/src/types.ts

```diff
@@ -273,6 +290,17 @@ export type FacetResult = Record<
   }
 >
 
+export type GroupResult =
+  | {
+      values: ScalarSearchableValue[]
+      result: Result[]
+    }[]
+  | {
+      values: ScalarSearchableValue[]
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      result: any
```

Comment:
No way to remove this `any`, right?

Reply:
resolved forwarding the type from SearchParams to Results.